kobo/apps/accounts/views.py

```diff
@@ -52,4 +52,15 @@ class SocialAccountViewSet(
     permission_classes = (IsAuthenticated,)
 
     def get_queryset(self):
-        return super().get_queryset().filter(user=self.request.user)
+        private_apps = [
```

Comment:
I don't think we need to exclude them here.

The environment api should return only public apps. If I'm a regular user, I don't know that Terraso has a private social app. 

But I login with a private social app, it's fine for me to see this. I could imagine a future where it lists all my social app connections and lets me disconnect any of them.

Reply:
@bufke Yeah, that makes sense. I'll remove this.